### PR TITLE
Update ECS task json schema

### PR DESCRIPTION
**Pull request by teastburn**
Received the below error when trying to use the `linuxParameters` setting then saw the kingpin jsonschema doesn't include it. So i updated from https://github.com/awslabs/amazon-ecs-intellisense-schema/blob/mainline/src/model/schema/schema.json and removed the `description` and `additionalFields` fields as they were not included before.

However, i do see that the error comes from `botocore` so I'm not entirely sure this fixes my issue.

```
23:23:25   CRITICAL  Unexpected exception caught! Please contact the author (Steve Mostovoy <smostovoy@nextdoor.com>) and provide them with this stacktrace
23:23:25   ERROR     [kingpin.actors.aws.ecs.Service] Parameter validation failed:
Unknown parameter in containerDefinitions[0]: "linuxParameters", must be one of: name, image, cpu, memory, memoryReservation, links, portMappings, essential, entryPoint, command, environment, mountPoints, volumesFrom, hostname, user, workingDirectory, disableNetworking, privileged, readonlyRootFilesystem, dnsServers, dnsSearchDomains, extraHosts, dockerSecurityOptions, dockerLabels, ulimits, logConfiguration
Traceback (most recent call last):
  File "/tmp/kingpin/kingpin/actors/base.py", line 491, in execute
    result = yield self.timeout(self._execute)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/base.py", line 329, in timeout
    deadline, fut, quiet_exceptions=(exceptions.ActorTimedOut))
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/aws/ecs.py", line 1341, in _execute
    yield self._ensure_service_present(service_name, existing_service)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/utils.py", line 72, in wrapper
    ret = yield gen.coroutine(f)(self, *args, **kwargs)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/aws/ecs.py", line 1303, in _ensure_service_present
    service_name, existing_service, override=override)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/utils.py", line 72, in wrapper
    ret = yield gen.coroutine(f)(self, *args, **kwargs)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/aws/ecs.py", line 998, in _update_service
    self.task_definition)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/utils.py", line 72, in wrapper
    ret = yield gen.coroutine(f)(self, *args, **kwargs)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/tmp/kingpin/tornado/gen.py", line 1021, in run
    yielded = self.gen.throw(*exc_info)
  File "/tmp/kingpin/kingpin/actors/aws/ecs.py", line 245, in _register_task
    self.ecs_conn.register_task_definition, **task_definition)
  File "/tmp/kingpin/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/tmp/kingpin/concurrent/futures/_base.py", line 398, in result
    return self.__get_result()
  File "/tmp/kingpin/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/tmp/kingpin/retrying.py", line 49, in wrapped_f
    return Retrying(*dargs, **dkw).call(f, *args, **kw)
  File "/tmp/kingpin/retrying.py", line 206, in call
    return attempt.get(self._wrap_exception)
  File "/tmp/kingpin/retrying.py", line 247, in get
    six.reraise(self.value[0], self.value[1], self.value[2])
  File "/tmp/kingpin/retrying.py", line 200, in call
    attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
  File "/tmp/kingpin/kingpin/utils.py", line 186, in wrapper
    return func(*args, **kwargs)
  File "/tmp/kingpin/kingpin/actors/aws/base.py", line 190, in thread
    return function(*args, **kwargs)
  File "/tmp/kingpin/botocore/client.py", line 253, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/tmp/kingpin/botocore/client.py", line 517, in _make_api_call
    api_params, operation_model, context=request_context)
  File "/tmp/kingpin/botocore/client.py", line 572, in _convert_to_request_dict
    api_params, operation_model)
  File "/tmp/kingpin/botocore/validate.py", line 270, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
ParamValidationError: Parameter validation failed:
Unknown parameter in containerDefinitions[0]: "linuxParameters", must be one of: name, image, cpu, memory, memoryReservation, links, portMappings, essential, entryPoint, command, environment, mountPoints, volumesFrom, hostname, user, workingDirectory, disableNetworking, privileged, readonlyRootFilesystem, dnsServers, dnsSearchDomains, extraHosts, dockerSecurityOptions, dockerLabels, ulimits, logConfiguration
```

### CODE REVIEW

File: kingpin/actors/aws/ecs.py [hunk 1]
```diff
@@ -361,7 +959,7 @@ def _list_task_definitions(self, status='ALL', family_prefix=None):
         raise gen.Return(task_definitions)
 
     @staticmethod
-    def _load_task_definition(task_definition_file, tokens, default_tokens={}):
+    def _load_task_definition(task_definition_file, tokens, default_tokens=None):
```
Comment (teastburn) [inline kingpin/actors/aws/ecs.py:962]: using `{}` in default params is a potential bug https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Comment (LaikaN57) [reply kingpin/actors/aws/ecs.py:962]: I feel like we used to have issues with setting this to `None` as well in the past.

Comment (teastburn) [reply kingpin/actors/aws/ecs.py]: It did make the line too long haha. Was that the issue?

**Comment by diranged** [comment]
@teastburn This all looks fine - but I don't know how much testing has gone into it. Do you want me to go ahead and merge it?

**Pull request closed without merging**